StackViewController/StackViewContainer.swift

```diff
@@ -271,7 +271,7 @@ public class StackViewContainer: UIView, UIScrollViewDelegate {
         precondition(index < items.endIndex)
         
         let item = items[index]
-        if items.count >= 1 && index == items.endIndex.predecessor() {
+        if items.count >= 1 && index == items.endIndex.predecessor() && index > 0 {
```

Comment:
Nah maybe it's fine to leave it. Assert only works in debug builds so stick with precondition.